terraform-use-modules/step2.md

```diff
@@ -71,8 +71,8 @@ registry](https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/2.
 Module outputs are usually either passed to other parts of your configuration,
 or defined as outputs in your root module. You will implement both uses in this tutorial.
 
-Open `outputs.tf`{{open}}. This file should contain the following.
+Open `outputs.tf`{{open}} and review the output variables defined.
```

Comment:
```suggestion
Open `outputs.tf`{{open}} and review the outputs defined in it.
```

Per [the docs](https://www.terraform.io/docs/language/values/outputs.html), it's either "output values" or "outputs".